components/eamxx/scripts/test-all-scream

```diff
@@ -64,17 +64,14 @@ OR
     parser.add_argument("-p", "--parallel", action="store_true",
                         help="Launch the different build types stacks in parallel")
 
-    parser.add_argument("-f", "--fast-fail", action="store_true",
```

Comment:
With `-f` gone, I was tempted to add `-f` as shortcut for `--force-baseline-regen`...